segregation/spatial/spatial_indexes.py

```diff
@@ -167,7 +185,9 @@ def calculate_etat(t):
         g_t_i = np.where(data.group_pop_var / data.total_pop_var >= t, True,
                          False)
         k = g_t_i.sum()
-        sub_delta_ij = delta[g_t_i,:][:,g_t_i]  # i and j only varies in the units subset within the threshold in eta_t of Hong (2014).
+        sub_delta_ij = delta[
```

Comment:
I think all the editings of the IDE is just fine. However, I think that for this `sub_delta_ij` it is better to keep with the original one because I think it is more informative without these extra spaces between the slices.